packages/docs/components/banner.md

```diff
@@ -0,0 +1,466 @@
+---
+template: component
+id: component-banner
+title: Banner
+description: Banners let users know important messages related to their overall experience on the website.
```

Comment:
for whatever reason the icon on the front page doesn't have this description text? Is there another vuepress build step required we need to run? Would be curious to know more about how this is wired together

![712056a ods dev_components_](https://user-images.githubusercontent.com/63716167/115885528-0aacc100-a41e-11eb-9747-e258240cf747.png)

Reply:
<img width="664" alt="Screen Shot 2021-04-23 at 9 33 51 AM" src="https://user-images.githubusercontent.com/36284167/115902212-0a5cf780-a417-11eb-9d81-565a04ebf837.png">

Fixed in the latest 🔨 

We're using frontmatter in the markdown to pass in `description` and `lead` along with a few other properties for each component.